denizen_scripts/survival/repo_link/shops/claim_upgrades_shop.dsc

```diff
@@ -46,6 +43,10 @@ claiming_protection_group_upgrades_events:
   type: world
   debug: false
   events:
+    on player opens claiming_protection_groyp_upgrades:
+      - define cost <script[claiming_system_upgrade_data].parsed_key[upgrades.claim_limit.cost]>
+      - define "lore:<&a>---------------------|<&e>Price<&co><&sp><&a><[cost]>|<&b>Right click while holding.|<&b>This will unlock <&a>10 <&b>more claims.|<&a>---------------------"
```

Comment:
```suggestion
      - define "lore:|:<&a>---------------------|<&e>Price<&co><&sp><&a><[cost]>|<&b>Right click while holding.|<&b>This will unlock <&a>10 <&b>more claims.|<&a>---------------------"
```